### ISSUE TITLE
Receive no  traffic

### ISSUE THREAD

**Issue by ubx** [roema/Open-Glider-Network-Groundstation#1]
Hi

Installed version 0.0.7 following the Easy Installation instruction. I have a FLARM (version 7.01) and a OGNTP device nearby:

- No traffic received
- Receive periodic UDP debugging messages (in my case over address  192.168.1.200)

![Screenshot from 2020-11-06 10-54-03](https://user-images.githubusercontent.com/516680/98437164-6ab5e200-20e0-11eb-81a5-6735b51e9f79.png)

BTW very interesting project!

**Comment by roema** [comment]
Hi Andreas!You need GPS fix.In picture, i see gnss zero.Wait for gps fix, gnss should show you the number of satellites.Then, decoding packets is possible and the groundstation is registering in ogn network. At this point, you should See aprs messages in port 12000.Please let me know, if it works vor not!Thanks! ManuAm 07.11.2020 10:21 schrieb Andreas Lüthi <notifications@github.com>:
Hi
Installed version 0.0.7 following the Easy Installation instruction. I have a FLARM (version 7.01) and a OGNTP device nearby:
No traffic receivedReceive periodic UDP debugging messages (in my case over address  192.168.1.200)

BTW very interesting project!

**Comment by roema** [comment]
Hi Andreas!

If i take a look at glider radar, i can see your station with last forwarded aircraft. Is your problem solved?

**Comment by ubx** [comment]
Hi Manu

I moved the ground station (Walpwil-M) outside and now it sees 7 satellites. I also turned off my other Raspi ground station (Walpwil), to avoid confusion. But still not traffic forwarded to the ogn network.

![Screenshot from 2020-11-07 10-45-06](https://user-images.githubusercontent.com/516680/98438088-e1a2a900-20e7-11eb-8208-69cba20756bc.png)

**Comment by roema** [comment]
Hi Andreas!

Please test to change "Ignore track bit" and "Ignore stealth bit" to true.

Manu

**Comment by ubx** [comment]
Done, still not traffic forwarded.

**Comment by roema** [comment]
Your ogn or flarm transponder has gps?

**Comment by roema** [comment]
Ok, no i can see you traffic. But there is a bug in ARPS coding.

Error, This is not a valid APRS packet: FLRC424E4>APRS,qAS,Walpwil-M:/104147h4703.70N00713.70En000/000/A=001555 !W94! id3EC424E4 +000fpm +0.0rot 82.00dB 0e -0.0kH

I will fix this today evening.

**Comment by ubx** [comment]
Thanks Manu

FYI on UDP debugging I can now see these messages:

![Screenshot from 2020-11-07 11-50-19](https://user-images.githubusercontent.com/516680/98439097-97bdc100-20ef-11eb-9769-9f10a093a13e.png)
![Screenshot from 2020-11-07 11-51-04](https://user-images.githubusercontent.com/516680/98439102-9b514800-20ef-11eb-94cd-5a57f5b5174d.png)

**Comment by roema** [comment]
I have found a BUG in APRS symbol table.
Can you test the attached firmware?
[ognbase.ino.esp32_test.zip](https://github.com/roema/Open-Glider-Network-Groundstation/files/5504685/ognbase.ino.esp32_test.zip)

Thanks, Manu

**Comment by ubx** [comment]
Hi Manu

Updated to the new firmware. No periodic UDP debugging messages anymore. Timestamp field doesn't change.

![Screenshot from 2020-11-07 13-25-01](https://user-images.githubusercontent.com/516680/98441427-76180600-20fe-11eb-868f-beeb3f637ac3.png)

**Comment by roema** [comment]
Have found a bug in aprs symbol table.
Can you test this image?

[ognbase.ino.esp32.zip](https://github.com/roema/Open-Glider-Network-Groundstation/files/5504758/ognbase.ino.esp32.zip)

 Manu

**Comment by roema** [comment]
Hi Andreas!

I habe uploaded wrong image, sorry. Please use this: 
[ognbase.ino.esp32.zip](https://github.com/roema/Open-Glider-Network-Groundstation/files/5504762/ognbase.ino.esp32.zip)

**Comment by ubx** [comment]
Hallo ManuIch bin im Moment nicht im "Labor".  Test heute Abend wieder.Grüsse Andreas 
-------- Ursprüngliche Nachricht --------Von: Manuel Rösel <notifications@github.com> Datum: 07.11.20  14:06  (GMT+01:00) An: roema/Open-Glider-Network-Groundstation <Open-Glider-Network-Groundstation@noreply.github.com> Cc: Andreas Lüthi <andreas.luethi@gmx.net>, Author <author@noreply.github.com> Betreff: Re: [roema/Open-Glider-Network-Groundstation] Receive no  traffic (#1) 
Hi Andreas!
I habe uploaded wrong image, sorry. Please use this:
ognbase.ino.esp32.zip

**Comment by roema** [comment]
> Hallo Manu Ich bin im Moment nicht im "Labor".  Test heute Abend wieder. Grüsse Andreas 
> […](#)
> -------- Ursprüngliche Nachricht --------Von: Manuel Rösel <notifications@github.com> Datum: 07.11.20 14:06 (GMT+01:00) An: roema/Open-Glider-Network-Groundstation <Open-Glider-Network-Groundstation@noreply.github.com> Cc: Andreas Lüthi <andreas.luethi@gmx.net>, Author <author@noreply.github.com> Betreff: Re: [roema/Open-Glider-Network-Groundstation] Receive no traffic (#1) Hi Andreas! I habe uploaded wrong image, sorry. Please use this: ognbase.ino.esp32.zip —You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.

Ok!

Vielen Dank!

**Comment by roema** [comment]
version0.0.8 should fix this problem

**Comment by ubx** [comment]
Hi Manu

With the latest image it works! But I'm a little confused. I thought the legacy protocol is for FLARM only.
If I set Protocol 1 to Legacy then both  OGNTP and FLARM stations are forwarded to the ogn network.

Andreas

**Comment by roema** [comment]
Hi Andreas!

C42464 is the id from your ogn basestation and used as callsign.
If you see a packet like 

```
Walpwil-M>APRS,TCPIP*,qAC,C42464:/203741h4703.67NI00713.70E&/A=001541
Walpwil-M>APRS,TCPIP*,qAC,C42464:/205125h4703.67NI00713.70E&/A=001541
Walpwil-M>APRS,TCPIP*,qAC,C42464:>205125h 0.0.8_ESP32 3.93V 
```

 its not an aircraft packet, that are status packets.

Are you sure that your station is sending OGNTP packets? If you've set legacy as protocol-1, that's actually impossible.

I will close the issue as the original problem has been resolved. However, I noticed a second problem in the APRS keepalive package which is fixed with version 0.0.10.

Let me know if your problem still exists !

Manu
